skyportal/handlers/api/photometric_series.py

```diff
@@ -0,0 +1,1764 @@
+        channel = self.get_query_argument('channel', None)
+        origin = self.get_query_argument('origin', None)
+        filename = self.get_query_argument('filename', None)
+        start_before = self.get_query_argument('startBefore', None)
+        start_after = self.get_query_argument('startAfter', None)
+        middle_before = self.get_query_argument('midBefore', None)
+        middle_after = self.get_query_argument('midAfter', None)
+        end_before = self.get_query_argument('endBefore', None)
+        end_after = self.get_query_argument('endAfter', None)
+        detected = self.get_query_argument('detected', None)
+        exp_time_exact = self.get_query_argument('expTime', None)
+        min_exp_time = self.get_query_argument('minExpTime', None)
+        max_exp_time = self.get_query_argument('maxExpTime', None)
+        min_frame_rate = self.get_query_argument('minFrameRate', None)
+        max_frame_rate = self.get_query_argument('maxFrameRate', None)
+        min_num_exp = self.get_query_argument('minNumExposures', None)
+        max_num_exp = self.get_query_argument('maxNumExposures', None)
+        instrument_id = self.get_query_argument('instrumentID', None)
+        followup_id = self.get_query_argument('followupRequestID', None)
+        assignment_id = self.get_query_argument('assignmentID', None)
+        owner_id = self.get_query_argument('ownerID', None)
+        mag_brighter = self.get_query_argument('magBrighterThan', None)
+        mag_fainter = self.get_query_argument('magFainterThan', None)
+        magref_brighter = self.get_query_argument('magrefBrighterThan', None)
+        magref_fainter = self.get_query_argument('magrefFainterThan', None)
+        max_rms = self.get_query_argument('maxRMS', None)
+        min_rms = self.get_query_argument('minRMS', None)
+        use_robust = self.get_query_argument('useRobustMagAndRMS', False)
+        min_median_snr = self.get_query_argument('minMedianSNR', None)
+        max_median_snr = self.get_query_argument('maxMedianSNR', None)
+        min_best_snr = self.get_query_argument('minBestSNR', None)
+        max_best_snr = self.get_query_argument('maxBestSNR', None)
+        min_worst_snr = self.get_query_argument('minWorstSNR', None)
+        max_worst_snr = self.get_query_argument('maxWorstSNR', None)
+        hash = self.get_query_argument('hash', None)
+        sort_by = self.get_query_argument('sortBy', 'obj_id')
+        sort_order = self.get_query_argument('sortOrder', 'asc')
+        page_number = self.get_query_argument('pageNumber', 1)
+        num_per_page = min(
+            int(self.get_query_argument("numPerPage", DEFAULT_SERIES_PER_PAGE)),
+            MAX_SERIES_PER_PAGE,
+        )
+
+        stmt = PhotometricSeries.select(self.current_user)
+
+        if ra is not None and dec is not None and radius is not None:
+            try:
+                ra = float(ra)
+                dec = float(dec)
+                radius = float(radius)
```

Comment:
Do some bounds checking here. Dont want to propagate further if they aren't correct.

Reply:
done